### PR TITLE
Write STEM image output to an HDF5 file

### PR DESCRIPTION
**Pull request by psavery**
This writes the "bright" and "dark" output to
an HDF5 file with paths "/stem/bright" and "/stem/dark",
respectively.

Signed-off-by: Patrick Avery <patrick.avery@kitware.com>

### CODE REVIEW

**Comment by psavery** [comment]
I moved it to a function. Let me know if I need to make any changes such as saving as 1D data with the dimensions saved as attributes.

File: examples/create_hdf5.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    Example: "python create_hdf5.py /path/to/data/data*.dat"
+
+    """
+
+    if len(files) == 0:
+        sys.exit('No files found')
+
+    reader = io.reader(files)
+    img = image.create_stem_image(reader, 160, 160,  40, 288);
```
[the remark this answers is not in the document]
Comment (psavery) [reply examples/create_hdf5.py]: Done.

**Comment by cjh1** [comment]
@psavery Works for me in testing. I merge master it the branch. Looks like there is just one more TODO to clean up.

**Comment by cjh1** [comment]
Also, could we add a quick markdown file documenting the format?

**Pull request merged**